java/src/jmri/implementation/DefaultRoute.java

```diff
@@ -1264,15 +1264,15 @@ public void run() {
                 final int toState = state;
                 final Turnout setTurnout = t;
                 ThreadingUtil.runOnLayoutEventually(() -> {   // eventually, even though we have timing here, should be soon
-                    setTurnout.setCommandedState(toState);
-                });         
+                    setTurnout.setCommandedStateAtInterval(toState); // delayed on specific connection by its turnoutManager
+                });
                 try {
-                    Thread.sleep(250 + delay);
+                    Thread.sleep(delay); // only the Route specific user defined delay is applied here
```

Comment:
This is a functional change that requires some serious thought.  People have conflguration files that assume there will be at least 250msec between Route firings, and this removes that.

Reply:
This replaces a fixed 250 minumum delay in code by a default delay of the same value, but user configurable to whatever is desired. Per Route and Per MatrixMastOutput delays will be added as configured.

Reply:
I understand (I think) that the user-configurable one starts at 250, but what about stored values in panel files?  If I've got 50 stored in a file, right now that means "wait 300" but after this change that'll mean "wait 50" which is quite different.

Reply:
I think the change two lines earlier attempts to ensure the delay remains, but I’m 100% on that.

Reply:
On opening an older config, finding no setting for the OutputDelay in the profile.xml, both the MatrixOutput Turnouts and SetRoute() commands will add a 250 mSec default delay between individual Turnout SetCommandedState instructions.

Reply:
@bobjacobsen was this concern resolved?

Reply:
I'm having trouble following the details of the conversation as presented, but I'm not sure it was resolved.

My concern is that people who already have delays set don't lose those when using this feature.